monitoring/angle-monitoring/src/main/java/com/farao_community/farao/monitoring/angle_monitoring/RaoResultWithAngleMonitoring.java

```diff
@@ -35,17 +36,23 @@ public RaoResultWithAngleMonitoring(RaoResult raoResult, AngleMonitoringResult a
         this.angleMonitoringResult = angleMonitoringResult;
     }
 
+    public AngleMonitoringResult getAngleMonitoringResult() {
+        return angleMonitoringResult;
+    }
+
     @Override
     public ComputationStatus getComputationStatus() {
-        if (angleMonitoringResult.isDivergent() || angleMonitoringResult.isUnknown()) {
-            return ComputationStatus.FAILURE;
-        } else {
+        if (Objects.isNull(angleMonitoringResult) || (!angleMonitoringResult.isDivergent() && !angleMonitoringResult.isUnknown())) {
```

Comment:
angleMonitoringResult mustn't be null : maybe require a non null value in constructor, for both objects

Reply:
I added the nullity check for angle monitoring result only